internal/version/version.go

```diff
@@ -232,8 +233,8 @@ func PercentageToCursor(pct int) RolloutBytes {
 	return cursor
 }
 
-// ShouldUpdate checks if for the the given rollout state, a user with the given nodeID should update.
-func ShouldUpdate(rollout Rollout, nodeID storj.NodeID) bool {
+// ShouldRollout checks if for the the given rollout state, a user with the given nodeID should update.
+func ShouldRollout(rollout Rollout, nodeID storj.NodeID) bool {
```

Comment:
Why renaming this method? What was wrong with "should update" based on rollout info and node id (the args)?

Reply:
This was done by Bryan if I don't remember wrong, so Michal may not know.

Reply:
yep

Reply:
I find it quite confusing. Can we rename it back to ShouldUpdate. This will also reduce the PR size a little bit.

Reply:
ok